### DISCUSSION TITLE
Why isn't  'T' and 'U' supported as pair of inputs for PropsSI?

### DISCUSSION THREAD

**Discussion by burtonrasmussen** [CoolProp/CoolProp#2413] [Q&A]
Hi, I'm new to using this library, and haven't been able to see where in the CoolProp website it mentions why I can't have this as an input for PropsSI. Here is the error code I got in response:

ValueError: This pair of inputs [TUmass_INPUTS] is not yet supported : PropsSI("P","T",393.15,"U",2628000,"water")

For more context, I'm using Spyder via the Anaconda installation of python. I'm trying to make a script that will take any 2 inputs(P,T,v,u,h,s,Q), then spit out P, T, v, u, h, s, and the state of the system. I have it working fine when I give it Pressure and temperature, or pressure and enthalpy, or pressure and specific volume. So that's why I'm wondering why this isn't a valid input for the function.

**Comment by ibell** [comment]
Because there can be multiple density solutions for T,U (and also T,H) inputs, and that is hard to deal with. Hence, CoolProp just doesn't support them. This is the first question about that in the last more than a decade, so I don't think too many people are needing T,U to justify the complexity
